optuna/samplers/_nsga2/sampler.py

```diff
@@ -124,9 +127,12 @@ def __init__(
                 " The interface can change in the future.",
                 ExperimentalWarning,
             )
+        if crossover_name not in ["uniform", "blxalpha", "sbx", "vsbx", "undx", "undxm", "spx"]:
+            raise ValueError(f"{crossover_name} is not exist in optuna.")
```

Comment:
Besides, IMO,
- it may be a bit more helpful to include the word "crossover" in the message
- the word "optuna" seems not to contain much information (so could be replaced with another word)
- => e.g., "Crossover '{crossover_name}' isn't supported by NSGAIISampler." seems more user friendly (IMO)